examples/gpbp_showcase.ipynb

```diff
@@ -27,29 +27,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Retrieving data for Timor-Leste of granularity level 0\n",
-      "Extracting geometry for Timor-Leste\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "adm_area = AdmArea(country=\"Timor-Leste\", level=0)"
+    "adm_area = AdmArea(country=\"Timor-Leste\", level=1)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "adm_area.get_adm_area(\"Timor-Leste\")"
+    "adm_area.get_adm_area(\"Baucau\")"
```

Comment:
I've changed the notebook so it only analyses one region in Timor-Leste. In this way it's much faster to run the full notebook on a smaller region. That's been helping me a lot in testing changes to the code.